README.md

```diff
@@ -1,3 +1,7 @@
 # End to End Benchmarking Scripts
 This repo contains performance test scripts which install, configure, execute and evaluate workloads.
 
+CI command to run all tests : ./ci/run_ci.sh  
```

Comment:
I would move this to a `README` in `/ci`.

Reply:
yes makes sense , will do the change